server/libmain/libmain.go

```diff
@@ -363,7 +364,7 @@ func StartAndRunServices(env environment.Env) {
 		StartGRPCServiceOrDie(env, buildBuddyServer, gRPCSPort, grpc.Creds(creds))
 	}
 
-	mux := http.NewServeMux()
+	mux := tracing.NewHttpServeMux()
```

Comment:
nit: could we take "delegate" as an argument to NewHttpServeMux rather than creating it inside the tracing package?

Reply:
Done.